### PR TITLE
week 6 content

### PR DESCRIPTION
**Pull request by rcyq**
Added links and content for week 6

### CODE REVIEW

File: contents/week6.html [hunk 1]
```diff
@@ -19,15 +13,54 @@ <h3 id="deadline1-content-week6" class="buttoned"><span>Deadlines </span></h3>
...
+                    <li>
+                        When published, read the CE1 peer code reviews in TEAMMATES.
+                    </li>
+                    <li>
+                        You should have received<br>
+                        (a) at least one peer code review AND<br>
+                        (b) comments about SLAP from your tutor. If that is not the case, please inform your tutor.
+                    </li>
+                    <li>
+                        Don&#39;t get upset if the reviews you received seem &#39;wrong&#39; or &#39;contradictory&#39;. If you are sure a point in the review is wrong, just ignore it. If you are not sure yourself, you can get the opinion of your team members before deciding to reject it.
```
Comment (damithc) [inline contents/week6.html]: Use normal chars instead of `&#39;` etc?

File: contents/week6.html [hunk 2]
```diff
@@ -37,51 +70,118 @@ <h3 id="things-to-do-content-week6" class="buttoned"><span>Things to do</span></
...
-            <h3> [Team, 2pts] Activity 1</h3>
+            <h3> [Team, 2pts] Define project tasks.</h3>
+            <div>
+                <ul>
+                    <li>
+                        <span style="text-decoration: underline">Re-schedule user stories</span> among milestones, if necessary. <br>
+                        You may wish to schedule &#39;risky&#39; features earlier so that you have enough time to recover if their implementation did not go according to expectations.
+                    </li>
+                    <li>
+                        <span style="text-decoration: underline">Assign components</span> in your architecture among team members. i.e. who will implement which component. If you have fewer components than team members, consider dividing bigger components into sub components.<br>
```
Comment (damithc) [inline contents/week6.html]: Remove inline styles. use css instead.

File: contents/week6.html [hunk 3]
```diff
@@ -90,30 +190,129 @@ <h3 id="ilo-content-week6" class="buttoned"><span>Intended Learning Outcomes</sp
...
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        [Humor <img src="https://lh3.googleusercontent.com/4h0URumHA0XPrlcVReuodEQgJngubOO5nsDlD8XwmHM12Pt0NGN1cvCtOtCjQSmSmAd0sLVE8CXol1C4kI0j8S2sQavo3TVz3XU_4ig3AueOiM3TOeB_DjpWxQPHXco8yp2U9kQ" style="width: 39.50px; height: 39.53px;">] <a target="_blank" href="https://www.google.com/url?q=http://www.youtube.com/watch?v%3DCDeG4S-mJts%26gl%3DSG%26hl%3Den-GB&sa=D&usg=AFQjCNF7gmmf3oAZ528nMg26HcYUAPdo9A">Hitler users Git</a>
+                         (warning: strong language), <a target="_blank" href="https://www.google.com/url?q=https://twitter.com/nakshayn/status/471887998603100160&sa=D&usg=AFQjCNERTytcE14E-aM1hixiCrbQlJo3Tg">When I see a door with a &#39;push&#39; sign...</a>
```
Comment (damithc) [inline contents/week6.html]: Limit line length to 110-120 chars. You can use 2 spaces for indenting.

**Comment by damithc** [comment]
Added some comments.

File: contents/week6.html (hunk 2 above)
Comment (rcyq) [inline contents/week6.html]: I can't seems to find a css which does underline only. Should I add it in common.css?

Comment (damithc) [inline contents/week6.html]: Sure, go ahead.

**Comment by rcyq** [comment]
Fixed the issues mentioned above.

File: contents/week6.html [hunk 4]
```diff
@@ -1,122 +1,437 @@
...
+      </div>
+    </div>
+    <div class="things-to-do things-to-do-week6">
+      <h3>Feedback for V0.0</h3>
+      <div>
+        <ul>
+          <li>
+            Read 
+            <a target="_blank" 
+            href="https://www.google.com/url?q=https://docs.google.com/document/d/1kAth-pTOzwCJFVjS6KU8O50bCvijDqpb6GsOyJTD0Ds/pub?embedded%3Dtrue&sa=D&usg=AFQjCNGcVznW6dBiriPohgxNHeOKAOH6Fg">
```
Comment (damithc) [inline contents/week6.html]: This link has some suffix and prefix added by Google docs. Can you remove them?

Comment (damithc) [inline contents/week6.html]: And in a few other places too.

**Comment by rcyq** [comment]
Removed additional suffix and prefix on the links

**Comment by damithc** [comment]
Some parts are messed up. Ensure that '+' and '++' buttons are working similar to week 2.
image

**Comment by rcyq** [comment]
Fixed the issue

**Comment by damithc** [comment]
These panel titles are not suitable. e.g. they don't tell the reader what the panel is about
image

**Comment by rcyq** [comment]
Updated panel titles

**Pull request merged**